packages/backend/server/src/plugins/copilot/storage.ts

```diff
@@ -31,6 +31,10 @@ export class CopilotStorage {
   ) {
     const name = `${userId}/${workspaceId}/${key}`;
     await this.provider.put(name, blob);
+    if (this.config.node.dev) {
```

Comment:
the reason?

Reply:
gpt vision need to query the image, when in debug env, its complax to proxy local dev env to a public domain, provide data url directly can bypass this problem